### PR TITLE
Support custom edge agent/hub image for connectivity test

### PR DESCRIPTION
**Pull request by philipktlin**
Support custom edge agent/hub image for connectivity test; therefore we can run release version/custom version of iotedge for connectivity test easily, e.g. 1.0.8 release version.

### CODE REVIEW

**Comment by philipktlin** [comment]
Please review and i am going to run tests; and post the result once completed.

File: test/connectivity/scripts/connectivityTest.sh [hunk 1]
```diff
@@ -52,6 +52,14 @@ function prepare_test_from_artifacts() {
     sed -i -e "s@<LogAnalyticsSharedKey>@$LOG_ANALYTICS_SHAREDKEY@g" "$deployment_working_file"
     sed -i -e "s@<UpstreamProtocol>@$UPSTREAM_PROTOCOL@g" "$deployment_working_file"
     
+    if [[ ! -z "$CUSTOM_EDGE_AGENT_IMAGE" ]]; then
+        sed -i -e "s@\"image\":.*azureiotedge-agent:.*\"@\"image\": \"$CUSTOM_EDGE_AGENT_IMAGE\"@g" "$deployment_working_file"
```
Comment (and-rewsmith) [inline test/connectivity/scripts/connectivityTest.sh]: Assuming this regex won't continue grabbing past the newline?

**Comment by and-rewsmith** [comment]
Let me know when test run complete please

File: test/connectivity/scripts/connectivityTest.sh (hunk 1 above)
Comment (philipktlin) [reply test/connectivity/scripts/connectivityTest.sh]: yes.

**Comment by philipktlin** [comment]
It works as expected, check this out https://msazure.visualstudio.com/One/_build/results?buildId=29014314&view=logs&j=6ecc137b-3a85-553a-1aab-7cdc03381ddc&t=8667053b-7599-4246-ab49-88e304c52961.

**Pull request merged**